src/universal/modules/meeting/components/MeetingUpdatesPrompt/MeetingUpdatesPrompt.js

```diff
@@ -1,36 +1,60 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import MeetingPrompt from 'universal/modules/meeting/components/MeetingPrompt/MeetingPrompt';
 import {createFragmentContainer} from 'react-relay';
+import HelpTextForTeam from 'universal/modules/meeting/components/HelpTextForTeam';
+import HelpTextMyRound from 'universal/modules/meeting/components/HelpTextMyRound';
+import MeetingPrompt from 'universal/modules/meeting/components/MeetingPrompt/MeetingPrompt';
 
 const MeetingUpdatesPrompt = (props) => {
   const {
+    agendaInputRef,
     localPhaseItem,
-    team: {teamMembers}
+    team
   } = props;
+  const {teamMembers} = team;
   const currentTeamMember = teamMembers[localPhaseItem - 1];
-  const heading = <span>{currentTeamMember.preferredName}, <i>what’s changed since our last meeting</i>?</span>;
+  const {isSelf: isMyMeetingSection, isCheckedIn} = currentTeamMember;
+  const isCheckedInFalse = Boolean(isCheckedIn === false);
+  const userHasCards = true;
```

Comment:
Ok @mattkrick this is what I need: Updates prompt knows about viewer/projects stubbed in here as `userHasCards` boolean.